scraper.py

```diff
@@ -0,0 +1,2 @@
+# Will scrape Cornell Dining's endpoint json and populate graphQL
```

Comment:
this is slightly misnamed -- scraping implies extracting data from HTML, you can just do this directly in `data.py` probably with the python `json` module

Reply:
+1 here, we don't need a `scraper.py`, it can be done in `data.py`